internal/services/consumer.go

```diff
@@ -10,11 +10,29 @@ import (
 	"context"
 	"log"
 	"os"
+	"strconv"
+	"time"
 
 	"github.com/Shopify/sarama"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
 	"github.com/strimzi/strimzi-canary/internal/config"
 )
 
+var (
+	recordsConsumed = promauto.NewCounter(prometheus.CounterOpts{
+		Name:      "records_consumed_total",
+		Namespace: "strimzi_canary",
+		Help:      "The total number of records consumed",
+	})
+	recordsLatency = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Name:      "records_latency",
+		Namespace: "strimzi_canary",
+		Help:      "Records end to end latency",
```

Comment:
```suggestion
		Help:      "Records end-to-end latency",
```

Are units implied, somehow, or should they be explicitly described?

Reply:
+1 for specifying it's in ms